Mono.Debugging/Mono.Debugging.Client/DebuggerSession.cs

```diff
@@ -1300,11 +1315,15 @@ internal protected void OnTargetDebug (int level, string category, string messag
 			}
 		}
 
-		internal protected void OnAssemblyLoaded (string assemblyLocation)
+		internal protected void OnAssemblyLoaded (Assembly assembly)
```

Comment:
This is an API break. Instead of changing the parameter, add a new overload that takes the Assembly as argument. The old overload can call the new one.